python-sdk/src/astro/sql/operators/append.py

```diff
@@ -88,18 +95,27 @@ def get_openlineage_facets(self, task_instance) -> OpenLineageFacets:
                         columns=self.columns,
                         metadata=self.source_table.metadata,
                     ),
-                    "schema_dataset_facet": SchemaDatasetFacet(
+                    "schema": SchemaDatasetFacet(
```

Comment:
Open lineage UI renders the facets based on https://github.com/OpenLineage/OpenLineage/blob/main/spec/OpenLineage.md#standard-facets.
To avoid naming conflicts, make sure that custom facets have a common prefix in their name related to the project that defines them: https://github.com/OpenLineage/OpenLineage/blob/main/spec/OpenLineage.md#custom-facet-naming

Reply:
Its as per the https://github.com/OpenLineage/OpenLineage/blob/main/spec/OpenLineage.md#standard-facets so that  it reflects on UI